tests/test_ascii.py

```diff
@@ -61,53 +63,55 @@ def test_create_set_message(self):
             self.settable_ascii_register.create_set_message(15)
         with pytest.raises(AttributeError):
             self.bad_settable_ascii_register = AsciiRegister(
-                commander=None,
+                component=None,
                 module_name="Bad",
                 module_id=0,
                 register_name="Bad",
                 read_only=False,
             )
 
+    @pytest.mark.skip("Not working.")
```

Comment:
Please create a ticket to make the tests that are marked as `skip` work and add TODOs in this class.

Reply:
I fixed the tests.